recipe/meta.yaml

```diff
@@ -747,7 +947,7 @@ outputs:
         # see https://github.com/apache/arrow/issues/37931
         - setuptools_scm <8
       run:
-        - {{ pin_subpackage('pyarrow', exact=True) }}
+        - {{ pin_subpackage('pyarrow-core', exact=True) }}
```

Comment:
This is interesting - i.e. the test suite passes without all underlying libraries available - but I think we should go the opposite way. We should test with `pyarrow-all`, to ensure maximum coverage of the test suite here.

Reply:
Done